src/server/api/root.ts

```diff
@@ -0,0 +1,16 @@
+import { createTrpcRouter } from "./trpc";
+import { authRouter } from "./routers/auth";
+import { exampleRouter } from "./routers/example";
+
+/**
+ * This is the primary router for your server.
+ *
+ * All routers added in /api/routers should be manually added here
+ */
+export const appRouter = createTrpcRouter({
+  example: exampleRouter,
+  auth: authRouter,
+});
+
+// export type definition of API
+export type AppRouter = typeof appRouter;
```

Comment:
I'm not a fan of this `root.ts`-file, I like index files contextually

The `_app.tsx` with the `_` is just to make sure it's alphabetically at the top